_posts/2022-08-23-New-series-From-the-editor's-desk.md

```diff
@@ -0,0 +1,71 @@
+| enable                   | turn on, activate, start            |
+| invalid                  | not valid                           |
+| primitive                | primitive data type, primitive type |
+| purge                    | delete, clear, remove               |
+| segregate                | separate, isolate                   |
+| trigger                  | initiate, invoke, launch, start     |
+| white day                | open day                            |
+| whitelist                | allow list                          |
+
+Version 2.0 replaced non-inclusive terminology (such as master, blacklist) throughout OpenSearch with inclusive terminology (such as cluster manager, allow list), but please let us know if you think we missed something so that we can address it in an expedient manner. Check out the associated [GitHub issue](https://github.com/opensearch-project/OpenSearch/issues/2589) for more information.
+
+## Accessibility
+
+Accessibility involves designing and creating websites, user interfaces, and documentation so that people with disabilities can perceive, navigate, and interact with them. OpenSearch follows basic accessibility guidelines to help ensure that our documentation is available and useful for everyone. Following these principles also helps improve the general usability of content.
+
+We follow these general accessibility guidelines in our documentation:
+
+* Links: Use link text that adequately describes the target page. For example, use the title of the target page instead of “here” or “this link.” In most cases, a formal cross-reference (the title of the page you're linking to) is the preferred style because it provides context and helps readers understand where they’re going when they choose the link.
+* Procedures: Not everyone uses a mouse, so use device-independent verbs; for example, use “choose” instead of “click.” 
+* Location: When you’re describing the location of something else in your content, such as an image or another section, use words such as “preceding,” “previous,” or “following” instead of “above” and “below.”
+
+We aim to follow these guidelines consistently, but, again, please let us know if you think we missed something, and we’ll work with you to make an appropriate change.
+
+## Join the conversation
+
+OpenSearch is committed to being a community where everyone can join and contribute, and we welcome community contributions to the [OpenSearch style guidelines](https://github.com/opensearch-project/documentation-website/blob/main/STYLE_GUIDE.md). To access our documentation, see the [OpenSearch documentation home page](https://opensearch.org/docs/latest). If you want to contribute to the OpenSearch documentation, see the [CONTRIBUTING.md](https://github.com/opensearch-project/documentation-website/blob/main/CONTRIBUTING.md) file, which covers how to open an issue or create a pull request on GitHub. We look forward to collaborating with you!
```

Comment:
Will you sign your name? Might be nice to have your name with your title. Unless it is on the side with your pic?

Reply:
My name and title will accompany the post.